app/models/user.rb

```diff
@@ -91,8 +91,12 @@ module Permissions
   ####################
   # Class method(s)  #
   ####################
-  def self.search_by_email(email)
-    User.where('lower(email) like ?', "#{email}%")
+  def self.search_by_email_or_real_name(email, real_name)
+    if email.nil?
+      User.where('lower(real_name) like ?', "#{real_name}%")
```

Comment:
If we change it to "%#{real_name}%" then it will match the input if it occurs anywhere in the real name field (allowing search for just last name, for example).